src/runtime_src/core/edge/drm/zocl/zocl_xclbin.c

```diff
@@ -752,21 +752,30 @@ zocl_xclbin_read_axlf(struct drm_zocl_dev *zdev, struct drm_zocl_axlf *axlf_obj,
 
 	/* Check unique ID */
 	if (zocl_xclbin_same_uuid(zdev, &axlf_head.m_header.uuid)) {
-		if (is_aie_only(axlf)) {
-			write_unlock(&zdev->attr_rwlock);
-			ret = zocl_load_aie_only_pdi(zdev, axlf, xclbin,
-			    client);
-			write_lock(&zdev->attr_rwlock);
-			if (ret)
-				DRM_WARN("read xclbin: fail to load AIE");
-			else {
-				zocl_create_aie(zdev, axlf, aie_res);
-				zocl_cache_xclbin(zdev, axlf, xclbin);
+		if ( !(axlf_obj->za_flags & DRM_ZOCL_FORCE_PROGRAM) )
```

Comment:
some c-style comment
1. no space between ( and !
2. it should be 
   if () {
   } else {
   }

Reply:
Sure @larry9523 , I will make the changes.

Reply:
Fixed the code.